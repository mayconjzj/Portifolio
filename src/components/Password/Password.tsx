import * as S from './styles';

export const Password = () => {
  return (
    <S.Content>
      <span>Sua senha gerada é:</span>
      <span>12345678</span>
      <span>Clique na senha para copiar.</span>
    </S.Content>
  );
};
