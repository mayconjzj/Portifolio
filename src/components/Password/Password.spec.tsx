import { render } from '@testing-library/react';

import { Password } from './Password';

describe('Password', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Password />);
    expect(baseElement).toBeTruthy();

    expect(baseElement).toMatchSnapshot();
  });
});
