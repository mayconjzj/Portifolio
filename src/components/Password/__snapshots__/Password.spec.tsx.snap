// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Password should render successfully 1`] = `
<body>
  <div>
    <div
      class="flex flex-col w-[200px]"
    >
      <span>
        Sua senha gerada eÃÅ:
      </span>
      <span>
        12345678
      </span>
      <span>
        Clique na senha para copiar.
      </span>
    </div>
  </div>
</body>
`;
